### DISCUSSION TITLE
InputBox prompt shows default message in ( )

### DISCUSSION THREAD

**Discussion by FloppyDisco** [microsoft/vscode-discussions#1435] [Extension Development QnA] [answered]
can anyone tell me how to get rid of the default prompt?

my apologies but i have not found anything relevant while searching. this is my first extension.

i have created an inputBox and i have set the title, prompt, and placeholder attributes. however, regardless of what i put in the prompt. it always displays the default message in parentheses. after the custom prompt. this is frustrating because the default message has nothing to do with the way the user will interact with my extension.
```
  const inputBox = vscode.window.createInputBox();
  inputBox.placeholder = "jump to ...";
  inputBox.prompt = "jumping";
  inputBox.title = "🐸";
```
<img width="792" alt="Screenshot 2024-08-20 at 2 46 02 PM" src="https://github.com/user-attachments/assets/695cd079-801c-43a5-b9e9-50e7b5e5beec">
how do i get rid of the "( press 'enter' to confirm or escape to cancel )"

thanks in advance.

**Comment by ArturoDent** [comment, marked as answer]
I don't think there is any way to get rid of it.  You'll probably have to file an issue.  But how do your users confirm or cancel the input?

**Reply by FloppyDisco** [reply]
it's really just the enter part, i would like to use the enter key for jumping to the next search match just like the native find widget.

**Reply by FloppyDisco** [reply]
i was just looking at your commits. it looks like we are more or less making the same tool. lol. yours looks much better than mine!
